pupil-spa/src/app/check/check.component.ts

```diff
@@ -291,8 +291,8 @@ export class CheckComponent implements OnInit {
       }
     }
 
-    // Set up the final page
-    this.allowedStates.push('complete');
+    // Set up the submission stage
+    this.allowedStates.push('submission-pending');
```

Comment:
I would expect a state after submission pending.  It doesn't look like there is one.

Reply:
@jon-shipley check component's last state is going to be submission pending.
Submission pending component is the connection link between the two possible last paths(check complete or submission failed).